system-tests/project-fixtures/vue/cypress.config.js

```diff
@@ -0,0 +1,10 @@
+import { defineConfig } from 'cypress'
```

Comment:
Do we need an entirely new project for this? Any chance we can add the test to an existing project?

Reply:
I like the pattern of having a `project-fixture` project that can cover multiple versions of a framework we support and a `mount.cy.js` that can test the behavior of the mount for that framework. We currently do not have this for `vue` so writing a test that can be shared is not possible. I def think we have too many of these projects and no clear pattern across them, but I'm bullish on this pattern. WDYT?